index.html

```diff
@@ -5330,49 +5329,85 @@ <h2>Service Privacy</h2>
+correlation of their <a>service endpoints</a> could become a powerful
+surveillance and inference tool.
       </p>
+<p>
+The variety of possible endpoints makes it particularly challenging to maintain 
+herd privacy, in which no information about the <a>DID subject</a> is leaked.
+</p>
+<p>
+First, because service endpoints may be specified as <a>URIs</a>, they could 
+unintentionally leak personal information because of the architecture of the service. 
+For example, a service endpoint of <code>http://example.com/MyFirstName</code> is leaking the term 
+<code>MyFirstName</code> to everyone who can access the DID Document. When linking to legacy systems, 
+this is an unavoidable risk and care should be taken in such cases. We encourage new, 
+DID-aware endpoints to use nothing more than the DID itself for any identification necessary,
+e.g., <code>http://example.com/did%3Aexample%3Aabc123</code>. Since that <code>did:example:abc123</code> is already 
+exposed in the DID Document, it leaks no additional information.
+</p>
+
+<p>
+Second, because a <a>DID document</a> can list multiple service endpoints, it is possible to 
+irreversibly associate services that are not associated in any other context. This 
+correlation on its own may lead to de-anonymization, revealing information about the 
+<a>DID subject</a> even if the <a>URIs</a> used did not.
+</p>
+<p>
+Third, because some types of <a>DID subjects</a> may be more or less likely to list specific 
+endpoints, e.g., a <a>DID</a> for an automobile may include a pointer to a public title 
+record at the Department of Motor Vehicles, while a <a>DID</a> for an individual would not.
+As such, the listing of a given service could, by itself, leak information that can 
+be used to infer something about the subject.
+</p>
+<p>
+It is the goal of herd privacy to ensure that the nature of specific subjects is 
+obscured by the population of the whole. To maximize herd privacy, implementers 
+need to rely on one &mdash; and only one &mdash; service endpoint, with that endpoint providing 
+a proxy or mediator service to further blind requests.
+</p>
+<p>
+Torward this end, consider any of the following:
+  <ul>
+  <li><strong>Negotiator Endpoint</strong> &mdash; Service for negotiating mutually 
+  agreeable communications channels, preferably using private set intersection. The
+    output of negotiation is a communication channel and whatever credentials may be 
+    needed to access it. </li>
+  <li><strong>Tor Endpoint</strong> (<a href="https://www.torproject.org/about/history/">Tor Onion Router</a>) &mdash;  
+  Provide a privacy-respecting address for reaching service endpoints. Any service that
+  can be provided online can be provided through TOR for additional privacy.</li>
+  <li><strong>Mediator Endpoint</strong> &mdash; 
+  <a href="https://hyperledger-indy.readthedocs.io/projects/hipe/en/latest/text/0036-mediators-and-relays/README.html">Mediators</a>
```

Comment:
This link will be better to reference a Mediator because it links to Aries (where the majority of work on agents has focused) and directly links to the latest commit updating the comment so some form of the content on the subject is always retrievable at the link provided.

https://github.com/hyperledger/aries-rfcs/blob/720bdab50e2d0437fda03028c1b17c69781bdd69/concepts/0046-mediators-and-relays/README.md